postgresql-simple.cabal

```diff
@@ -82,6 +82,7 @@ library
       base              >=4.6.0.0  && <4.16
     , bytestring        >=0.10.0.0 && <0.12
     , containers        >=0.5.0.0  && <0.7
+    , exceptions        >=0.10.4   && <0.11
```

Comment:
All `with*` functions in `posgresql-simple` are just `IO`. e.g. [`withTransaction`](https://hackage.haskell.org/package/postgresql-simple-0.6.4/docs/Database-PostgreSQL-Simple-Transaction.html#v:withTransaction).

Reply:
Fixed